prometheus/operator/base/jupyterhub-servicemonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: jupyterhub
+    team: opendatahub
+  name: jupyterhubservicemonitor
+spec:
+  endpoints:
+    - port: 8080-tcp
+  selector:
+    matchLabels:
+      app: jupyterhub
```

Comment:
Would this result in the service monitor attempting to scrape itself or something due to the labels?

Reply:
it looks for services with that label